maintenance/wikia/getDatabase.php

```diff
@@ -69,11 +73,19 @@
 		$pattern = '/city_id: (\d+), cluster: c([1-9])/';
 		preg_match($pattern, $page, $matches);
 		$city_id = $matches[1];
-		$cluster = strtr($matches[2], '123456', 'ABCDEF');
-		$databaseDirectory = "database_$cluster";
-		if ($cluster == 'F') {
-			// FIXME
-			$databaseDirectory = "database-f";
+		// Don't use fetched cluster number if it was provided on script invoke params list
+		if ( empty( $clusterNumberParam ) ) {
+			$clusterNumberParam = $matches[2];
+		}
+		if ( $clusterNumberParam > 9 ) {
+			echo "Clusters higher than 9 are not yet operated by this script. Time to update the script.\n";
+			exit;
+		}
+		$clusterLetter = strtr( $clusterNumberParam, '123456789', 'ABCDEFGHI' );
```

Comment:
The script already looks up the cluster from wikicities database, I don't think we ever need to provide it on the command line? 

The FIXME here was related to the directory name.   Cluster A-E use underscore but cluster F uses a dash.  We can also have a mapping of cluster -> separator like:

 $separator = strtr($matches[2], '1234567', ‘______-‘);